### PR TITLE
Add summaries and urls

### PR DESCRIPTION
**Pull request by mshadbolt**
Add summaries and urls to the schema so that is renders more nicely on the docs website, see: https://candig.github.io/CanDIGv2/technical/ingest/operations/ingest_operationsupdate_role/

### CODE REVIEW

File: ingest_openapi.yaml [hunk 1]
```diff
@@ -1,11 +1,15 @@
 openapi: 3.0.0
+servers:
+  - url: https://CANDIG_URL/ingest
```
Comment (daisieh) [inline ingest_openapi.yaml]: Do we have an automated way to interpolate this variable? I'd rather leave it blank if we don't.

**Review by daisieh — APPROVED** [review]
LGTM, but I'm not 100% sure about adding urls to the document, unless there's an automated way to put those in there with the correct domains.

**Comment by mshadbolt** [comment]
If I don't put a full url in there, it populates with the raw github url, which I think is worse? 
Screenshot 2024-10-31 at 12 32 47 PM

**Review by daisieh — APPROVED** [review]
LGTM

**Pull request merged**